custom_components/zidoo/media_player.py

```diff
@@ -349,6 +397,17 @@ async def async_set_audio(self):
         """Sets or toggles the audio_track subtitle."""
         await self.coordinator.player.set_audio()
 
+    async def async_send_command(self, command: Iterable[str], **kwargs: Any) -> None:
```

Comment:
What is the reasoning behind adding the `async_send_command` with the limited command list in the media player?   We currently have this in the remote entity along with a more complete list using the ZKEYS.   With the remote we get a command list in the dialog properties.